### PR TITLE
FIX increase tolerance for numerical errors in some tests

### PR DESCRIPTION
**Pull request by Badr-MOUFAD**
Fixes language warnings.

(To get CI rights)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,9 +50,9 @@
...
           install_requires=[
-            "celer>=0.6.1",
-            "download", "hyperopt",
-            "libsvmdata", "matplotlib>=2.0.0", "numba",
-            "numpy>=1.12", "scipy>=0.18.0",
-            "scikit-learn>=0.21", "seaborn>=0.7", ]
+              "celer",
+              "download", "hyperopt",
+              "libsvmdata", "matplotlib>=2.0.0", "numba",
+              "numpy>=1.12", "scipy>=0.18.0",
```
[the remark this answers is not in the document]
Comment (Badr-MOUFAD) [reply setup.py]: according to the last log 

https://github.com/QB3/sparse-ho/runs/6862358854?check_suite_focus=true#step:5:233

we should specify the version of numpy to be 1.21 or less

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -23,9 +23,10 @@ jobs:
       run: |
         pip install -U pip
         eval "$(conda shell.bash hook)"
-        conda env create --file environment.yml python=3.8
+        conda env create --file environment.yml
         conda activate sparse-ho-env
         pip install -e .
+        pip install celer --no-binary
```
Comment (Badr-MOUFAD) [inline .github/workflows/build.yml:29]:
```suggestion
        pip install celer --no-binary all
```

File: environment.yml [hunk 3]
```diff
@@ -16,3 +16,4 @@ dependencies:
         - cvxpy
         - torch
         - cvxpylayers
+        - git+https://github.com/mathurinm/celer.git
```
Comment (mathurinm) [inline environment.yml:19]: remove

File: examples/plot_lassoCV.py [hunk 4]
```diff
@@ -62,7 +62,7 @@
 t0 = time.time()
 reg = LassoCV(
     cv=kf, verbose=True, tol=tol, fit_intercept=False,
-    alphas=alphas, max_iter=1e5).fit(X, y)
+    alphas=alphas, max_iter=100_000).fit(X, y)
```
[the remark this answers is not in the document]
Comment (Badr-MOUFAD) [reply examples/plot_lassoCV.py:65]: this is not for cosmetics,

This breaks down the test as ``max_iter`` must be int and not ``float``

[the remark this answers is not in the document]
Comment (mathurinm) [reply examples/plot_lassoCV.py:65]: nice catch !

**Comment by mathurinm** [comment]
Doc build fails but not related to this PR https://app.circleci.com/pipelines/github/QB3/sparse-ho/1018/workflows/584448b6-b849-4cfc-96ea-4d21abff6433/jobs/1027

merging

**Pull request merged**